src/test_helpers.rs

```diff
@@ -18,15 +18,6 @@ cfg_if! {
                 .expect("DATABASE_URL must be set in order to run tests")
         }
 
-        use diesel::serialize::Output;
-        use diesel::sql_types::TypeMetadata;
-        /// Returns a `Output` suitable for testing `ToSql` implementations.
-        /// Unsafe to use for testing types which perform dynamic metadata lookup.
-        pub fn create_testing_output<DB: TypeMetadata>() -> Output<'static, Vec<u8>, DB> {
-            use std::mem;
-            #[cfg_attr(feature = "clippy", allow(invalid_ref))]
-            Output::new(Vec::new(), unsafe { mem::uninitialized() })
```

Comment:
This line causes https://asan.saethlin.dev/ub?crate=diesel_geometry&version=1.4.0

Reply:
This is not a fix, I just stripped out some tests because they were causing issues for me when I slapped this port together